{
  "Fault": {
    "faultcode": "Client",
    "faultstring": "An exception has been raised as a result of client data.",
    "detail": {
      "Errors": {
        "ErrorDetail": {
          "Severity": "Hard",
          "PrimaryErrorCode": {
            "Code": "151044",
            "Description": "No tracking information available"
          }
        }
      }
    }
  }
}
